srbhr/Resume-Matcher
# send_to_parser_dir method bug fix

## Pull Request Title


## Description


## Type


- [ x] Bug Fix
- [ x] Code Refactoring

## Proposed Changes


- Implemented feature to check for existing file in directory and if so, change file name by adding timestamp before moving file to directory.

## How to Test


1. Run flask --app run command in terminal
2. Go to website and enter data. Primarily only needs a resume
3. Hit submit and it should function normally
4. Then do step 2 - 3 again and there should be no errors as well as see the same file but with a timestamp added to the resume directory

## Checklist


- [ x] The code compiles successfully without any errors or warnings
- [ x] The changes have been tested and verified
